# Created at 2023-06-09T11:08:33.815
org.testng.TestNGException: 
Two tests in the same suite cannot have the same name: Testng
	at org.testng.TestNG.checkTestNames(TestNG.java:1057)
	at org.testng.TestNG.sanityCheck(TestNG.java:1044)
	at org.testng.TestNG.run(TestNG.java:1108)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)


